maintainers/maintainer-list.nix

```diff
@@ -8444,6 +8444,12 @@
     githubId = 3747396;
     name = "Nathan Isom";
   };
+  NelsonJeppesen = {
```

Comment:
Please, add yourself to maintainer list in separate commit.

Reply:
Done